### DISCUSSION TITLE
Sugestão para Fichas e recompensas

### DISCUSSION THREAD

**Discussion by jonathangl** [YuriAlessandro/gerador-ficha-tormenta20#368] [Ideias]
Bom dia, tenho uma ideia para a futura pagina de criação de fichas de personagens, fazer um avanço e retrocesso de níveis, assim pode ser criado um personagem de nível 20 passando por todos os níveis, com todas as habilidades planejadas e ir avançando a ficha conforme a aventura vai seguindo, isso facilita o uso da mesma ficha para outras campanhas sem precisar perder tempo criando uma nova ficha.

A outra ideia é para a tabela de recompensas, o mestre ter a opção de adicionar o resultado da rolagem física do dado.

**Comment by YuriAlessandro** [comment]
Estamos trabalhando num gerador de ficha que você pode ir colocando a ficha do nível 1 ao 20.

Ao contrário acho difícil de fazer.

Eu vou trabalhar num novo gerador de recompensas em breve, dando uma boa melhorada nele.

**Comment by YuriAlessandro** [comment]
Com a nova update isso é possível
